docs/reference/run.md

```diff
@@ -1590,6 +1590,23 @@ For example, you can specify either `/foo` or `foo` for a `host-src` value.
 If you supply the `/foo` value, Docker creates a bind mount. If you supply
 the `foo` specification, Docker creates a named volume.
 
+The `z` and `Z` modifiers will change the SELinux labels on the mount points.
+The `z` option will make the files available to any container, using the `s0` label,
+while the `Z` option will label the files with the same label as the container, so
+that they are exclusive to that container.
+
+WARNING Use of :z or :Z can have impacts on your system if used too broadly.
+For example, the use of these options against a users HOME directory or other top
+level system directory would result in the relabeling of those directories for use
+by containers. There are some blacklisted paths, but this could cause issues such
+as being unable to SSH back into a system. When using the relabel options, it should
+only be used on directories that are intended for use by containers, and not the
+host system itself. In most cases, such as shared access,  it is better to label
+in advance. If your container does require broader access to system directories,
+then use of '--security-opt label:disable' with the 'docker run' command is a better
```

Comment:
Is this such a good thing to encourage?

Reply:
I swear we already have something about this in the docs, let me find it. Yes, here: https://docs.docker.com/engine/admin/volumes/bind-mounts/#configure-the-selinux-label Perhaps you can just link to that instead of putting these docs here.

Reply:
oh, forgot about that; good point. @justincormack perhaps you can look at that section in the docs and see if anything is missing there? Then we can link to there